app/utility/tasks.py

```diff
@@ -0,0 +1,55 @@
+from celery import Celery
```

Comment:
So still forming thoughts on organization of the celery components but for the actual tasks, what do you think about making the package/dir/module path very explicit and discrete.

Basically if Im calling a task from somewhere in caldera Id like to know exactly why this call is going somewhere else and what is the 'tasks' module.

-->tl;dr, maybe something like `app.utility.celery_tasks.planning` for the tasks module with all planning functions in it. So if we are calling it from planning service its more clear that we dipping into a celery task function to handle it. Thoughts?